docs/destination.md

```diff
@@ -92,3 +92,102 @@ The Timeplus source plugin for Airbyte is in the early stage. Please contact us
 ## Trigger Actions via webhook{#webhook}
 
 You can also add automations to trigger other systems to take actions when Timeplus finds any real-time insights. Simply choose the **Webhook** as the action type and optionally set a message body (by default, the entire row will be encoded as a JSON document and sent to the webhook). You can use this approach to perform rule-based automation without human interaction, such as swapping a overheated equipment, scaling up to scaling down the server farm, or reminder users on slack, etc. Please check [this blog](https://www.timeplus.com/post/build-a-real-time-security-app-in-3-easy-steps) for real-world examples.
+
+
+## Sink API
+In case calling API to create a sink, here are the reference
+
+### kafka
+
+refet to [https://kafka.apache.org/](https://kafka.apache.org/)
+
+| Property                | Required    | Description                                               | Default |
+|-------------------------|---------|-----------------------------------------------------------|---------|
+| brokers                   | yes | Specifies the list of broker addresses. This is a comma-separated string. such as `kafka1:9092,kafka2:9092,kafka3:9092`| | |
+| topic                     | yes | Specifies the Kafka topic to send data to.              | |
+| batch_count               | no  | Specifies the Kafka offset configuration.    support `latest,earliest`            |`1000`|
+| data_type                 | yes | Specifies the data type to use for creating the stream.   support `json`,` `,`protobuf` | | 
+| sasl                      | no  | Specifies the Simple Authentication and Security Layer (SASL) mechanism for authentication. support `none`,`plain`,`scram-sha-256`,`scram-sha-512` | `none` | |
+| username                  | no  | Specifies the username for authentication.               ||
+| password                  | no  | Specifies the password for authentication.               ||
+| tls.disable               | no  | If set to `true`, disables TLS encryption.               |`false`|
+| tls.skip_verify_server    | no  | If set to `true`, skips server certificate verification when using TLS. |`false`|
+| protobuf_schema           | no  | Specifies the Protobuf schema.                          |
```

Comment:
May need to mention we need the full text of schema, not just a path or URI

Reply:
as suggested by Calvin, we plan remove protobuf in the doc